src/ExExpression.elm

```diff
@@ -475,10 +491,14 @@ operators =
     , ( "/", "/" )
     , ( ">>", ">>>" )
     , ( "<|", "<<<" )
-    , ( "%", "" ) -- Exception
-    , ( "//", "" ) -- Exception
-    , ( "rem", "" ) -- Exception
-    , ( "^", "" ) -- Exception
+    , ( "%", "" )
+      -- Exception
+    , ( "//", "" )
+      -- Exception
+    , ( "rem", "" )
+      -- Exception
+    , ( "^", "" )
+      -- Exception
```

Comment:
Should move the comments above, not below. Not sure if it's elm format or sth else